backend/danswer/bots/slack/constants.py

```diff
@@ -0,0 +1,2 @@
+LIKE_BLOCK_ACTION_ID = "feedback-like"
```

Comment:
we already have constants for these, why does it need to be different for slack?